packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/SQLEditor.tsx

```diff
@@ -112,13 +112,23 @@ export function SQLEditor({ table, isLiveQuery, setIsLiveQuery }: Props) {
               <div ref={containerRef} className="min-h-[21px] w-full">
                 <Editor
                   width="100%"
-                  theme="hc-black"
+                  theme="vs-dark"
                   value={decodeURIComponent(field.value)}
                   options={monacoOptions}
                   language="sql"
                   onChange={(value) => field.onChange(encodeURIComponent(value ?? ""))}
-                  onMount={(editor) => {
+                  onMount={(editor, monaco) => {
                     editorRef.current = editor;
+                    monaco.editor.defineTheme("custom-vs-dark", {
+                      base: "hc-black",
+                      inherit: true,
+                      rules: [{ token: "string.sql", foreground: "#C5947C" }],
```

Comment:
is this needed anymore?

Reply:
Yeah, for some reason still needed, otherwise it's red like before. I have a guess that web-based monaco editor has a limited theme support, or there is some other kind of difference.

Reply:
is there a way to get a reference to the hex value from the original theme rather than hardcoding it here?

Reply:
would be better agreed, couldn't find the way